packages/geo/src/lib/layer/layer-legend/layer-legend.component.ts

```diff
@@ -145,10 +141,11 @@ export class LayerLegendComponent implements OnInit, OnDestroy {
           return err;
         }
       })
-    ).subscribe((legend: string) => {
-      this.legendGraphic[this.currentItemTitle] = legend;
-      this.cdRef.detectChanges();
-    });
+      ).subscribe(function(item, legendGraph) {
```

Comment:
I think something like this would be cleaner (arrow function are equivalent) : 

).subscribe(obs => {
        const idx = this.legendItems$.value.findIndex(leg => leg.title === item.title);
        const legendGraph = obs as string;
        this.legendItems$.value[idx].imgGraphValue = legendGraph;
        this.cdRef.detectChanges();
});

Reply:
Change done. Thank @PhilippeLafreniere18